.clang-tidy

```diff
@@ -0,0 +1,8 @@
+---
+# https://clang.llvm.org/extra/clang-tidy/checks/list.html
+Checks: <-
+  clang-diagnostic-*,
+  clang-analyzer-*',
+  -clang-analyzer-valist.Uninitialized,
```

Comment:
This is how you remove a check. Unfortunately, these two gave a lot of false positives in the few source files I checks.

valist.Uninitialized: https://bugs.llvm.org/show_bug.cgi?id=41311
security.insecureAPI.DeprecatedOrUnsafeBufferHandling: It doesn't like snprintf(). Wants me to use snprintf_s(). What is that?

Reply:
I am not familiar with that, maybe you need to undefine some of these???

As with all bounds-checked functions, printf_s , fprintf_s, sprintf_s, and snprintf_s are only guaranteed to be available if __STDC_LIB_EXT1__ is defined by the implementation and if the user defines __STDC_WANT_LIB_EXT1__ to the integer constant 1 before including <stdio.h>.

https://en.cppreference.com/w/c/io/fprintf

Reply:
I guess these aren't supported by glibc? Thanks for the link